core/backend/src/test/standalone/RenderMaterialElement.test.ts

```diff
@@ -86,15 +86,11 @@ describe("RenderMaterialElement", () => {
     return textureId;
   }
 
-  describe("insert", () => {
-    it("with default values", () => {
-      test({}, {});
-    });
-
+  describe("read", () => {
     it("should be able to convert TextureId to hexadecimal string outside javascript's double precision range of 2^53", async () => {
       // LargeNumericRenderMaterialTextureId is a database with a single RenderMaterial whose jsonProperties look something like:
-      // {"materialAssets":{"renderMaterial":{"Map":{"Diffuse":{"TextureId":9223372036854775808}}}}}
-      // 9223372036854775808 is equivalent to 2^63 which is equivalent to the hexadecimal string 0x8000000000000000.
+      // {"materialAssets":{"renderMaterial":{"Map":{"Diffuse":{"TextureId":9223372036854775807}}}}}
+      // 9223372036854775807 is equivalent to 2^63 - 1 which is equivalent to the hexadecimal string 0x7fffffffffffffff.
```

Comment:
It seems to me that since this would pass with the broken native code, making this change is actually detrimental to the purpose of this test, which is to verify that large values work.

Reply:
9223372036854775807 actually doesn't pass on intel without the changes that paul suggested on the C++ side. It would give me back 0x800000 on intel due to the floating point conversion stuff.

Reply:
I didn't realize that this now fails on Intel without any native fixes. I'm guessing that means that it fails on both Intel and Arm, and if that is the case, then I agree that this change is good, and you can resolve this conversation.

Reply:
I think it might've passed on ARM, but not 100% sure. I added a couple more large numbers to my test database in hopes thats a bit more thorough.

Reply:
All tests pass (on native PR, which also runs the TS tests in this PR). @tcobbs-bentley please resolve your comment if satisfied.